jetbrains-ultimate/src/software/aws/toolkits/jetbrains/services/lambda/nodejs/NodeJsLambdaHandlerResolver.kt

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.lambda.nodejs
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.psi.NavigatablePsiElement
+import com.intellij.psi.PsiElement
+import com.intellij.psi.search.GlobalSearchScope
+import software.aws.toolkits.jetbrains.services.lambda.LambdaHandlerResolver
+
+class NodeJsLambdaHandlerResolver : LambdaHandlerResolver {
+
+    override fun version(): Int = 1
+
+    // TODO
+    override fun findPsiElements(
```

Comment:
https://github.com/aws/aws-toolkit-jetbrains/blob/feature/nodejs-support/jetbrains-ultimate/src/software/aws/toolkits/jetbrains/ultimate/services/lambda/nodejs/NodeJSLambdaHandlerResolver.kt ?

Reply:
Yeah, I know the existence of this feature branch. It's just it is a bit too old and we have migrated to Lambda packager to SAM, that's why I want to start a new branch. Also, the Lambda Handler Resolver is not sophisticated enough. Last, it does not have any test with it.

This CR is simply focusing on the Runtime Group and Lambda Builder implementation with integration tests. I will provide other parts and tests one by one to be easier for review.

Reply:
As I mentioned in the below comment, this implementation is not sophisticated and with no tests. I will provide a separate CR for this specifically.